rskj-core/src/main/java/co/rsk/rpc/modules/eth/EthModuleTransactionBase.java

```diff
@@ -57,54 +56,37 @@ public EthModuleTransactionBase(Constants constants, Wallet wallet, TransactionP
 
     @Override
     public synchronized String sendTransaction(Web3.CallArguments args) {
-        Account account = this.wallet.getAccount(new RskAddress(args.from));
-        String s = null;
+        
+    	Account senderAccount = this.wallet.getAccount(new RskAddress(args.from));
+        String txHash = null;
+        
         try {
-            String toAddress = args.to != null ? ByteUtil.toHexString(stringHexToByteArray(args.to)) : null;
```

Comment:
it was.

Reply:
because its been done inside the TransactionArgumentsUtil.processArguments where also the transactionPool maybe called. And the transactionPool is syncronized

Reply:
yes I see, but whats the purpose of this sync? I'm not really too deep into it

Reply:
Probably because of the storage